lib/ddtrace/contrib/dalli/quantize.rb

```diff
@@ -1,17 +1,17 @@
+require 'ddtrace/contrib/dalli/ext'
+
 module Datadog
   module Contrib
     module Dalli
       # Quantize contains dalli-specic quantization tools.
       module Quantize
-        MAX_CMD_LENGTH = 100
-
         module_function
 
         def format_command(operation, args)
           placeholder = "#{operation} BLOB (OMITTED)"
           command = [operation, *args].join(' ').strip
           command = Utils.utf8_encode(command, binary: true, placeholder: placeholder)
-          Utils.truncate(command, MAX_CMD_LENGTH)
+          Utils.truncate(command, Ext::QUANTIZE_MAX_CMD_LENGTH)
```

Comment:
👍

Reply:
Side note: would be great if we could do utf8_encode and truncate in the same "operation" so that we would not need to create temporary string of possibly unbounded length.  But its not a scope of this PR

Reply:
Yeah, interesting point. Consider opening an issue to track this suggestion.